pkg/drivers/hyperv/powershell_windows.go

```diff
@@ -26,15 +25,13 @@ func cmd(args ...string) error {
 	return err
 }
 
-func parseLines(stdout string) []string {
-	resp := []string{}
-
-	s := bufio.NewScanner(strings.NewReader(stdout))
-	for s.Scan() {
-		resp = append(resp, s.Text())
+func getFirstLine(stdout string) string {
```

Comment:
Not part of the util.go ?

Reply:
This package is the only user of `getFirstLine` so I kept it there as it's a bit specialized.

Reply:
the issue in question was the only place for `resp[0]`, for which we also use the `| get-unique` operator in powershell. I don't mind, but found it very generic.

Reply:
ok I moved it.